integration/examples/test-plugin/gcb/skaffold.yaml

```diff
@@ -10,10 +10,10 @@ build:
     context: ./leeroy-app/
     plugin: 
       name: docker
+      properties:
+        target: targetStage
   executionEnvironment:
     name: googleCloudBuild
-    properties:
```

Comment:
why are we removing this?

Reply:
I figured it would be easier for people to test this example if skaffold guessed the project from the image name instead of having to change this field in the skaffold.yaml every time

Reply:
👍